src/DependencyInjection/EventBusExtension.php

```diff
@@ -37,6 +38,10 @@ protected function loadInternal(array $mergedConfig, ContainerBuilder $container
 
         if ($mergedConfig['logging']['enabled']) {
             $loader->load('event_bus_logging.yml');
+
+            $container->getDefinition('simple_bus.event_bus.notifies_message_subscribers_middleware')
+                ->replaceArgument(1, new Reference('logger'))
+                ->replaceArgument(2, '%simple_bus.event_bus.logging.level%');
```

Comment:
I don't think so, but I can't easily remove it now since maybe someone else is using it :)